src/clients/private/_components/BottomAction/BottomActionBarButton.tsx

```
@@ -0,0 +1,36 @@
+                    <div className="flex gap-1 text-gray-400 bg-gray-200 btn btn-sm no-animation btn-animation hover:!bg-gray-150">
+                        {Icon}
+                        {buttonText}
+                    </div>
+                </Tippy>
+            ) : (
+                <button
+                    className={`flex gap-1 btn btn-sm no-animation btn-animation btn-white ${
+                        isPending ? 'link_to-loading' : ''
+                    }`}
+                    onClick={onClick}
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)